Jasmine/Tetris/ViewModels/TetrisViewModel.swift

```diff
@@ -1,3 +1,14 @@
+import Foundation
+
 class TetrisViewModel: TimedGameViewModel {
+    /// Specifies the total time allowed in the game.
+    var totalTimeAllowed: TimeInterval
+
+    /// Specifies the time remaining in the game.
+    var timeRemaining: TimeInterval
 
+    init() {
+        totalTimeAllowed = 0
+        timeRemaining = 0
+    }
```

Comment:
In this case there's a `timeRemaining` which should be the same as `totalTimeAllowed`, but such logic/assignment should really be handled within the timer class.
Also, not being able to use `extension TetrisViewModel: TimedGameProtocol` makes it harder to tell which variables are the ones from `TimedGameProtocol`, and we would have to adopt the same technique for other things like sound etc, so the main vm class would be conforming to all of them directly instead of through extension, which would be quite messy imo.